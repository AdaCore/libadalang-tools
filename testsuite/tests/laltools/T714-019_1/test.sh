#!/bin/sh
outgoing_calls -P p.gpr -S ./src/main.adb -L 2 -R 11
outgoing_calls -P p.gpr -S ./src/main.adb -L 4 -R 15
outgoing_calls -P p.gpr -S ./src/main.adb -L 4 -R 4
outgoing_calls -P p.gpr -S ./src/main.adb -L 27 -R 4
