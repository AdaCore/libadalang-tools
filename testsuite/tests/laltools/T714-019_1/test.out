<SubpDecl ["Foo"] p.ads:3:4-3:32> <Id "Foo" main.adb:7:7-7:10>
<SubpDecl ["Foo"] p.ads:3:4-3:32> <Id "Foo" main.adb:13:25-13:28>
<SubpDecl ["Foo"] p.ads:3:4-3:32> <Id "Foo" main.adb:17:25-17:28>
<SubpDecl ["Foo"] p.ads:3:4-3:32> <Id "Foo" main.adb:19:15-19:18>
Node is not a name.
Node is not a subprogram.
Node is not a defining name.
